src/bootstrap/bootstrap.py

```diff
@@ -540,11 +554,14 @@ def download_toolchain(self):
 
             # Unpack the tarballs in parallle.
             # In Python 2.7, Pool cannot be used as a context manager.
-            p = Pool(min(len(tarballs_download_info), cpu_count()))
+            pool_size = min(len(tarballs_download_info), get_cpus())
+            print('pool size', pool_size)
```

Comment:
nit:
```suggestion
            if verbose:
                print('pool size', pool_size)
```

Reply:
My bad, it was just debug output for me that made it in the PR, but I will change the wording and make it gated on verbose.